pkg/machines/components/vmDisksTabLibvirt.jsx

```diff
@@ -64,7 +64,7 @@ class VmDisksTabLibvirt extends React.Component {
             return null;
         }
 
-        if (vm.status === 'running') {
+        if (vm.state === 'running') {
```

Comment:
There's a similar thing here:

    pkg/kubernetes/scripts/virtual-machines/components/VmsListingRow.jsx:    const phase = (vm.status && vm.status.phase) || _("n/a");

but there `vm.status` seems to be an object, while here it's a string. Are these two different `status`es really?

Reply:
I am not familiar with k8s package, I am not sure if it's problem that there as well.

For the commit here we need to check the vm.state string as set here, https://github.com/cockpit-project/cockpit/blob/master/pkg/machines/libvirt.es6#L1048

vm.status is clearly a typo, not initialized anywhere.

Reply:
Indeed, thanks! Sorry, I totally missed the /kubernetes, I just saw the "machines" there :)